packages/babel-helper-remap-async-to-generator/src/index.js

```diff
@@ -83,12 +83,15 @@ function classOrObjectMethod(path: NodePath, callId: Object) {
 
   node.async = false;
 
-  const container = t.functionExpression(null, [], t.blockStatement(body.body), true);
+  const container = t.functionExpression(node.key, [], t.blockStatement(body.body), true);
   container.shadow = true;
   body.body = [
     t.returnStatement(t.callExpression(
-      t.callExpression(callId, [container]),
```

Comment:
You don't seem to have mentioned this change in your description anywhere. What's the goal of this?